tests/drivers/build_all/sensors_a_h.conf

```diff
@@ -31,4 +31,5 @@ CONFIG_HMC5883L=y
 CONFIG_HP206C=y
 CONFIG_HTS221=y
 CONFIG_ADXL345=y
+CONFIG_MAX17055=y
```

Comment:
Please move this to `tests/drivers/build_all/sensors_i_z.conf`

Reply:
done